--- origsrc/SDL2_mixer-2.0.2/configure.in	2017-12-08 00:10:41.332303600 -0600
+++ src/SDL2_mixer-2.0.2/configure.in	2017-12-08 00:10:50.539073200 -0600
@@ -248,7 +248,10 @@ if test x$enable_music_mod = xyes -a x$e
             *-*-darwin*)
                 modplug_lib=[`find_lib libmodplug.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                modplug_lib=/usr/bin/cygmodplug-1.dll
+                ;;
+            *-*-mingw32*)
                 modplug_lib=[`find_lib "libmodplug*.dll"`]
                 ;;
             *)
@@ -315,7 +318,10 @@ return 1;
             *-*-darwin*)
                 mikmod_lib=[`find_lib libmikmod.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                mikmod_lib=/usr/bin/cygmikmod-3.dll
+                ;;
+            *-*-mingw32*)
                 mikmod_lib=[`find_lib "libmikmod*.dll"`]
                 ;;
             *)
@@ -399,7 +405,10 @@ AC_HELP_STRING([--enable-music-midi-flui
                 *-*-darwin*)
                     fluidsynth_lib=[`find_lib libfluidsynth.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    fluidsynth_lib=/usr/bin/cygfluidsynth-1.dll
+                    ;;
+                *-*-mingw32*)
                     fluidsynth_lib=[`find_lib "fluidsynth*.dll"`]
                     ;;
                 *)
@@ -476,7 +485,10 @@ if test x$enable_music_ogg = xyes; then
                 *-*-darwin*)
                     ogg_lib=[`find_lib libvorbisfile.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    ogg_lib=/usr/bin/cygvorbisfile-3.dll
+                    ;;
+                *-*-mingw32*)
                     ogg_lib=[`find_lib "libvorbisfile*.dll"`]
                     ;;
                 *)
@@ -535,7 +547,10 @@ if test x$enable_music_flac = xyes; then
                 *-*-darwin*)
                     flac_lib=[`find_lib libFLAC.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    flac_lib=/usr/bin/cygFLAC-8.dll
+                    ;;
+                *-*-mingw32*)
                     flac_lib=[`find_lib "libFLAC-*.dll"`]
                     ;;
                 *)
@@ -577,7 +592,10 @@ if test x$enable_music_mp3 = xyes -a x$e
             *-*-darwin*)
                 smpeg_lib=[`find_lib libsmpeg2.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                smpeg_lib=/usr/bin/cygsmpeg2-2-0-0.dll
+                ;;
+            *-*-mingw32*)
                 smpeg_lib=[`find_lib "smpeg2*.dll"`]
                 ;;
             *)
@@ -639,7 +657,10 @@ if test x$enable_music_mp3_mpg123 = xyes
             *-*-darwin*)
                 mpg123_lib=[`find_lib libmpg123.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                mpg123_lib=/usr/bin/cygmpg123-0.dll
+                ;;
+            *-*-mingw32*)
                 mpg123_lib=[`find_lib "libmpg123*.dll"`]
                 ;;
             *)
