--- origsrc/SDL2_mixer-2.0.1/configure.in	2017-07-06 00:09:44.214031600 -0500
+++ src/SDL2_mixer-2.0.1/configure.in	2017-07-06 00:09:59.122022900 -0500
@@ -335,7 +335,10 @@ return 1;
             *-*-darwin*)
                 mikmod_lib=[`find_lib libmikmod.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                mikmod_lib=/usr/bin/cygmikmod-3.dll
+                ;;
+            *-*-mingw32*)
                 mikmod_lib=[`find_lib "libmikmod*.dll"`]
                 ;;
             *)
@@ -420,7 +423,10 @@ AC_HELP_STRING([--enable-music-midi-flui
                 *-*-darwin*)
                     fluidsynth_lib=[`find_lib libfluidsynth.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    fluidsynth_lib=/usr/bin/cygfluidsynth-1.dll
+                    ;;
+                *-*-mingw32*)
                     fluidsynth_lib=[`find_lib "fluidsynth*.dll"`]
                     ;;
                 *)
@@ -499,7 +505,10 @@ if test x$enable_music_ogg = xyes; then
                 *-*-darwin*)
                     ogg_lib=[`find_lib libvorbisfile.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    ogg_lib=/usr/bin/cygvorbisfile-3.dll
+                    ;;
+                *-*-mingw32*)
                     ogg_lib=[`find_lib "libvorbisfile*.dll"`]
                     ;;
                 *)
@@ -559,7 +568,10 @@ if test x$enable_music_flac = xyes; then
                 *-*-darwin*)
                     flac_lib=[`find_lib libFLAC.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    flac_lib=/usr/bin/cygFLAC-8.dll
+                    ;;
+                *-*-mingw32*)
                     flac_lib=[`find_lib "libFLAC-*.dll"`]
                     ;;
                 *)
@@ -602,7 +614,10 @@ if test x$enable_music_mp3 = xyes -a x$e
             *-*-darwin*)
                 smpeg_lib=[`find_lib libsmpeg2.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                smpeg_lib=/usr/bin/cygsmpeg2-2-0-0.dll
+                ;;
+            *-*-mingw32*)
                 smpeg_lib=[`find_lib "smpeg2*.dll"`]
                 ;;
             *)
